src/commands/partner.ts

```diff
@@ -1,13 +0,0 @@
-import { Command } from 'commander';
-import Partner from '../lib/Partner';
-
-export default function partner(program: Command) {
-    const partner = new Command('partner').description('Partner Commands');
```

Comment:
Remove this command from readme file

Reply:
Aslo raise jira ticket to update partner documentation

Reply:
Are we not handling proper deprecation message to user if they tries to hit this cmd?